src/eval/mod.rs

```diff
@@ -75,6 +77,18 @@ pub fn print(args: Vec<Expr>, env: &mut Env) -> Result<Expr, SpressoError> {
     Ok(result)
 }
 
+pub fn input(args: Vec<Expr>, env: &mut Env) -> Result<Expr, SpressoError> {
+    match print(args, env) {
+        Ok(_) => (),
+        Err(err) => return Err(err),
+    };
+    let mut buffer = String::new();
+    let stdin = io::stdin();
+    stdin.read_line(&mut buffer).unwrap();
```

Comment:
It would be nicer to return an Spresso error here instead of panicking and exiting on `.unwrap()`.